javascripts/application.js

```diff
@@ -88,14 +88,10 @@ $(document).ready(function() {
           "</ul>" +
         "</nav>";
 
+      // support inline-nav css with a class in the <body> tag
       $('body').addClass('with-inline-navigation');
-      var $table = $(".usa-layout-docs-main_content h1:first-child+.table-wrapper");
-      if ($table.length > 0){
-        $table.after(inlineNavigation);
-      } else {
-        $("main > p").first().after(inlineNavigation);
-      }
```

Comment:
Wondering if it could be something like 
`$("main  h2").first().before(inlineNavigation)`

I haven't tried it locally or anything though

Reply:
With that change, if there's a lot of opening text above the `<h2>` it pushes the nav box pretty far down the page on desktop. Willing to go with that but would like opinion re: this screenshot before I push that change.
 
![2021-11-05 15 53 39 localhost 8d342459124f](https://user-images.githubusercontent.com/814887/140577244-43577a48-a1e1-461d-ac01-03675a4181e1.png)

Reply:
Thanks for pointing this one out, I didn't realize the introductions get that long and it's good to see a more extreme case. Testing it out, it still seems to make sense to put it after the intro, even if it's a longer one, so that there is at least some description of the page content and the introductory content doesn't feel like it's missing from the subnav .

Reply:
Agreed! Change is pushed. I checked several pages, including ones without h2 tags, ones that start with a table (e.g. "Chicago"), as well as typical informational pages. Everything looks good to my eye.

Reply:
Awesome! The pages that I checked look good to me. The federalist preview comes up 404 for the Chicago page, so can't comment on the table one.